src/maintainer/conda_forge_yml.rst

```diff
@@ -23,6 +23,7 @@ Top-level fields
 
 * :ref:`appveyor`
 * :ref:`azure-config`
+* :ref:`self-hosted_azure-config`
```

Comment:
I guess this is a leftover. As mentioned earlier we don't need an extra key in conda-forge.yaml to set it up, so let's remove this.
```suggestion
```